lib/TH/generic/THTensor.c

```diff
@@ -510,6 +510,28 @@ void THTensor_(squeeze1d)(THTensor *self, THTensor *src, int dimension)
   }
 }
 
+int THTensor_(isTransposed)(const THTensor *self)
```

Comment:
1) I'm not sure if this code is correct in general. I think what you're asking is "is this a non-contiguous tensor with no holes". Is that what you meant? I think in that case it works except for a corner case when you have a stride of 0. For example,

```
t = torch.Tensor(10,1,32)
u = t:select(2,0):expand(10,32)
```
In this case you have size=[10,32] and stride=[32,0]. It does have holes.

2) This function is never used anywhere. Can you remove it? Since it's not used anywhere I don't know what it's semantics should be.